pkg/envoy/resources.go

```diff
@@ -60,7 +59,7 @@ var (
 )
 
 // HandleResourceVersionAck is required to implement ResourceVersionAckObserver.
-func (cache *NPHDSCache) HandleResourceVersionAck(ackVersion uint64, nackVersion uint64, node *envoy_api_v2_core.Node, resourceNames []string, typeURL string, detail string) {
+func (cache *NPHDSCache) HandleResourceVersionAck(ackVersion uint64, nackVersion uint64, nodeIP string, resourceNames []string, typeURL string, detail string) {
```

Comment:
Add a comment about what an empty `nodeIP` string signifies since callers can pass an empty string here.

Reply:
Callers of this never pass an empty `nodeIP`. I added a comment to explain what the function does, though.